src/components/app.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+
+import Header from "./blocks/Header";
+import RandomPlanet from "./blocks/RandomPlanet";
+import ItemList from "./blocks/ItemList";
+import PersonDetails from "./blocks/PersonDetails";
+import AppBackground from "./blocks/AppBackground";
+import ButtonToggle from "./controls/ButtonToggle";
+
+import "./style.css";
+
+class App extends Component {
+  state = {
+    randomPlanetIsOpen: true,
+  };
+
+  handleClickToggle = () => {
+    this.setState((state) => {
+      return {
+        randomPlanetIsOpen: !this.state.randomPlanetIsOpen,
+      };
+    });
+  };
+
+  render() {
+    const randomPlanet =
+      this.state.randomPlanetIsOpen ? (
+      <RandomPlanet />
+    ) : null;
+
+    return (
+      <>
+        <div className='app'>
+          <Header />
+          {randomPlanet}
```

Comment:
`const randomPlanet =
      this.state.randomPlanetIsOpen ? (
      <RandomPlanet />
    ) : null;`

Имеет ли смысл выносить это выражении  в отдельную константу? Может проще использовать `&&` (Ниже пример)
```suggestion
          {this.state.randomPlanetIsOpen && <RandomPlanet />}
```

Reply:
да, хорошая идея